styleguide/components/21-atoms/inputs/inputs.config.json

```diff
@@ -1,4 +1,4 @@
 {
   "title": "Inputs",
-  "order": 7
+  "hidden": true
```

Comment:
Why hide the input fields? Are we only going to show them inside their respective molecules?

Reply:
Yes, I don't want to cause confusion by showing input fields without labels.
The same reason we hide 'figcaption'.. they just cannot be used by themselves.

Now if you search 'input' the first thing you see is something which should not be used...

Do agree we should carefully document the molecules to explain which parts are optional and which not. But I'd leave that for the documentation ticket.

Reply:
Ok agreed, this seems like a good approach.